Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealitySceneSystemProfile.asset

```diff
@@ -0,0 +1,84 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_GeneratorAsset: {fileID: 0}
+  m_Script: {fileID: 11500000, guid: b81dae1d46879b444aa9847f7961649f, type: 3}
+  m_Name: DefaultMixedRealitySceneSystemProfile
+  m_EditorClassIdentifier: 
+  isCustomProfile: 1
+  useManagerScene: 1
+  managerScene:
+    Name: DefaultManagerScene
+    Path: Assets/MixedRealityToolkit.SDK/StandardAssets/Scenes/DefaultManagerScene.unity
+    Included: 1
+    BuildIndex: 0
+    Tag: Untagged
+    Asset: {fileID: 3671201678493226958, guid: ae7bb08d297fb69408695d8de0962524, type: 3}
```

Comment:
The SceneInfo struct stores path and scene name data, so when the object goes missing it can still find it.

If you click Search for missing scene does it find one in your project? Or is it totally blown away? I included DefaultManagerScene in the SDK folder but maybe its GUID got omitted.

Reply:
It does find it when I click that button! The `fileID` changes